soc/arm/ti_k3/j721e/soc.c

```diff
@@ -0,0 +1,81 @@
+/* Copyright (C) 2023 BeagleBoard.org Foundation
+ * Copyright (C) 2023 S Prashanth
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/init.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/fatal.h>
+
+#include <zephyr/drivers/interrupt_controller/intc_vim.h>
+
+static int j721e_init(void)
+{
+	return 0;
+}
+
+uint32_t sys_clock_elapsed(void)
+{
+	return 0;
+}
+
+uint32_t sys_clock_cycle_get_32(void)
+{
+	return 0;
+}
+
+uint64_t sys_clock_cycle_get_64(void)
+{
+	return 0;
+}
```

Comment:
how are you keeping the time?

Reply:
systick support is not added.

Reply:
how are you managing delays or the time in general? like `k_msleep()`?

Reply:
I am working on the timer support.

Reply:
ok, but please, next time just clarify in the PR message that this is a non-working port otherwise you will end up pissing maintainers off by forcing them to review something not yet complete and working.

Reply:
Thank You, I will update the PR message.